test/test-network/systemd-networkd-tests.py

```diff
@@ -389,6 +390,15 @@ def test_sit_tunnel(self):
         self.assertTrue(self.link_exits('dummy98'))
         self.assertTrue(self.link_exits('sittun99'))
 
+    def test_isatap_tunnel(self):
+        self.copy_unit_to_networkd_unit_path('12-dummy.netdev', '25-isatap-tunnel.netdev', 'isatap.network')
+        self.start_networkd()
+
+        self.assertTrue(self.link_exits('dummy98'))
+        self.assertTrue(self.link_exits('isataptun99'))
+        output = subprocess.check_output(['ip', '-d', 'link', 'show', 'isataptun99']).rstrip().decode('utf-8')
+        self.assertRegex(output, "isatap")
```

Comment:
Please add a whitespace after "isatap", that is, use "isatap ". Otherwise this returns true even if `ISATAP=false` as the device name includes "isatap".

Reply:
true :)